extensions/blocks/calendly/index.js

```diff
@@ -15,6 +16,7 @@ import icon from './icon';
  */
 import './editor.scss';
 
+export const REGEX = /(?<!\.)calendly\.com.+?([^"']*)/i;
```

Comment:
We can't use look behinds. Depending on what we want to match in this instance I think we can change it to the other version:

```
export const REGEX = /(^|\/\/)calendly\.com.+?([^"']*)/i;
```